AK/IPv4Address.h

```diff
@@ -159,6 +159,8 @@ struct Traits<IPv4Address> : public GenericTraits<IPv4Address> {
 #ifdef KERNEL
 template<>
 struct Formatter<IPv4Address> : Formatter<ErrorOr<NonnullOwnPtr<Kernel::KString>>> {
+    static constexpr bool is_debug_only() { return false; }
```

Comment:
We're already `TRY`'ing the failure away, this one can just drop `ErrorOr<>` from the template.

Reply:
It passes a guaranteed non-error value to the ErrorOr formatter, which will then defer to the regular formatter. I'm not sure why that is, but it's definitely not needed :)